packages/cli-repl/src/format-output.ts

```diff
@@ -249,7 +250,12 @@ function formatListCommands(
 
 export function formatError(error: Error, options: FormatOptions): string {
   let result = '';
-  if (error.name) result += `\r${clr(error.name, 'mongosh:error', options)}: `;
+  if (error.name) {
+    result += `\r${clr(error.name, 'mongosh:error', options)}`;
+    const codeName = (error as CodedError).codeName;
+    if (codeName) result += `\r[${codeName}]`;
```

Comment:
```suggestion
    if (codeName) result += `[${codeName}]`;
```

right? otherwise the `\r` here would return the cursor back to the start of the line, overwriting the `error.name` part added before it

Reply:
yes, I noticed now that I was adding tests 😅